### DISCUSSION TITLE
not able to complete github student pack github classroom assigment

### DISCUSSION THREAD

**Discussion by sujanbimali9** [community/community#73394] [GitHub Education]
### Hi everyone,

![Screenshot from 2023-11-04 19-25-11](https://github.com/community/community/assets/137367456/df37ab90-bd95-4a86-ba6b-f2fe0aef6494)

**Comment by JHermosillaD** [comment]
Hello @sujanbimali9 , you have already finished the _experience-primer-copilot_ of the **Primer: Copilot path**. Unfortunately, once the README.md is updated, you cannot go back. However, from your pull request generated in your repository, it is shown that you still need to finish steps 2, 3, 4, and 5.

Then, my recommendation is that:
1. Verify your active GitHub Copilot account by clicking [here](https://github.com/features/copilot) or reading the [general instructions](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-individuals). Activate for the first time if necessary.
2. Return to your [repository](https://github.com/Exp-Primer-Copilot-Cohort-1/experience-primer-copilot-sujanbimali9/tree/b16688d9bb00cfd43fba693e34e7e57b79bb9cf8) and check if you have an active codespace. If not, create a new one.
3. Read the instructions  2, 3, 4, and 5 carefully and perform steps again. You can see each step in the [history](https://github.com/Exp-Primer-Copilot-Cohort-1/experience-primer-copilot-sujanbimali9/commits/main/README.md) of your README.md. Just click on the **_view at this point in the history_** button of each commit (i.e. of each step).
4. Once you finish all the steps, delete the codespace from the same menu from which you activated it.
5. Verify that you don't have any pull requests that need to be merged.
6. Wait a couple of minutes and try to validate the status of your [learning path](https://education.github.com/experiences/primer_copilot) trying to mark it as completed again.

I hope I have made it clear.
Regards!

**Comment by OliverB21** [comment]
I'm having the same issue [here](https://github.com/Exp-Primer-Copilot-Cohort-1/experience-primer-copilot-OliverB21). I've completed all the steps and it's marking it at 5/6, can't see why as it's not returning anything
